docs/usage.md

```diff
@@ -114,9 +114,17 @@ gcc main.c libbar.so libbar_shim.so libmain_shim.so -Wl,-z,now \
     -I $IA2_INCLUDE_DIR -fPIC -Wl,-rpath=. -Wl,-T/$REPO_ROOT/libia2/padding.ld
 ```
 
-Shared objects that are assigned one a trusted protection key must have certain
+All read-only and relro sections of shared objects will be shared with all
```

Comment:
super minor nit: s/shared objects/DSOs/ since "shared" means two different things in this sentence.

Reply:
Done. I wanted to avoid DSO here because that gets confusing when the main binary is PIE, but hopefully its better now.